examples/applications/langchain-chat/python/langchain_chat.py

```diff
@@ -201,12 +201,13 @@ def create_chain(
 class LangChainChat(Processor):
 
     def init(self, config):
-        self.openai_key = config.get("openai-key", "")
-        self.astra_db_token = config.get("astra-db-token", "")
-        self.astra_db_id = config.get("astra-db-id", "")
-        self.astra_keyspace = config.get("astra-db-keyspace", "")
-        self.astra_table_name = config.get("astra-db-table", "")
 
+        # the values are configured in the resources section in configuration.yaml
+        self.openai_key = config.resources.openai.get("access-key", "");
```

Comment:
I'm surprised you can reference `config.resources`. Shouldn't it be `config["resources"]` ?

Reply:
you are right